curriculum/challenges/english/25-front-end-development/workshop-merge-sort/6569b4e0bd29d17d4b53b16c.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 6569b4e0bd29d17d4b53b16c
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
+# --description--
+
+Now, you are going to replicate the same `while` loop logic for `right_part`.
```

Comment:
I think we can combine this step with the following ones and create the while loop in one step. They would have just made the other one and it would be relatively easy to complete it.